Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "Convex"
+uuid = "f65535da-76fb-5f13-bab9-19810c17039a"
+
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MathProgBase = "fdba3010-5040-5b88-9595-932c9decdf73"
+OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+
+[compat]
+MathProgBase = "0.7"
+OrderedCollections = ">=1.0.0"
+julia = ">=0.7"
```

Comment:
Should be `"0.7, 1.0"`

Reply:
It should work fine with 1.1 and nightly (1.2) as well though.

Reply:
`1.0` means `1.x`

Reply:
Read https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1